proto/pki/proposed_certificate_revocation.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package zigbeealliance.distributedcomplianceledger.pki;
+
+option go_package = "github.com/zigbee-alliance/distributed-compliance-ledger/x/pki/types";
+
+import "cosmos_proto/cosmos.proto";
+
+message ProposedCertificateRevocation {
+  string subject = 1; 
```

Comment:
as an option you may consider here to make an embedded composition with `CerificateIdentifier` like [here](https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/4b152049bf86e7d6dfc83183664a90826177dc9f/proto/dclauth/account.proto#L16).

Generally it wouldn't affect code with getters, for sure code with constructors would be impacted. [Example](https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/4b152049bf86e7d6dfc83183664a90826177dc9f/x/dclauth/keeper/msg_server_propose_add_account.go#L67-L68).